core/src/test/scripts/test_data/study_various_issues/result_report.html

```diff
@@ -299,7 +291,7 @@ <h4 class="modal-title" id="gridSystemModalLabel">Listing all values for the mes
                             <div class="modal-content">
                               <div class="modal-header">
                                 <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                                <h4 class="modal-title" id="gridSystemModalLabel">Listing all values for the message: Line will not be loaded due to the variant classification filter. Filtered types: [Silent, Intron, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR, RNA]</h4>
+                                <h4 class="modal-title" id="gridSystemModalLabel">Listing all values for the message: Line will not be loaded due to the variant classification filter. Filtered types: [Silent, Intron, 3&#39;UTR, 3&#39;Flank, 5&#39;UTR, 5&#39;Flank, IGR, RNA]</h4>
```

Comment:
is it correct that all non-alpha characters are encoded?

Reply:
This is done by the validator itself. Tests pass so I guess its no problem.